SPECS/xml-commons-apis/xml-commons-apis.spec

```diff
@@ -37,7 +37,10 @@ Source4:        https://repo1.maven.org/maven2/xml-apis/xml-apis-ext/1.3.04/xml-
 BuildRequires:  ant
 BuildRequires:  fdupes
 BuildRequires:  java-devel
+BuildRequires:  javapackages-filesystem
```

Comment:
Since we already have an [SRPM published for this package](https://packages.microsoft.com/cbl-mariner/2.0/preview/extended/srpms/), we should also bump the `Release` tag.